sklego/metrics.py

```diff
@@ -0,0 +1,19 @@
+import numpy as np
+
+
+def correlation_score(column):
+    """
+    The correlation score can score how well the estimator predictions
+    correlate with a given column. This is especially usefull to use
+    in situations where "fairness" is a theme.
+    :param column: Name of the column (when X is a dataframe)
+    or the index of the column (when X is a numpy array).
+    :return: Negative correlation between estimator.predict(X) and X[colum]
+    (in gridsearch, larger is better and we want to typically punish correlation).
+    """
+    def fairness_metric(estimator, X, y_true=None):
+        """Remember: X is the thing going *in* to your pipeline."""
+        if isinstance(X, np.ndarray):
```

Comment:
how about:

```
sensitive_col = x[:, column] if isinstance(X, np.ndarray) else x[column]
return -np.abs(np.corrcoef(estimator.predict(X), sensitive_col)[1, 0])
```
to get rid of some duplication?

Reply:
done